notebooks/llm-chatbot/llm-chatbot.ipynb

```diff
@@ -13,28 +13,34 @@
     "While a decent intent-based chatbot can answer basic, one-touch inquiries like order management, FAQs, and policy questions, LLM chatbots can tackle more complex, multi-touch questions. LLM enables chatbots to provide support in a conversational manner, similar to how humans do, through contextual memory. Leveraging the capabilities of Language Models, chatbots are becoming increasingly intelligent, capable of understanding and responding to human language with remarkable accuracy.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.            pipe.generate(new_prompt, config, streamer)
</pre><p>I believe it should look differently. we need to demonstrate how to use pipe.start_chat() and pipe.end_chat() and also history maintenance on pipeline side with model state representation, please check this example <a href="https://github.com/openvinotoolkit/openvino.genai/blob/3f206d2c50bdd6d6d35ef13c40d90451b1746ef0/text_generation/causal_lm/cpp/chat_sample.cpp" target="_blank">https://github.com/openvinotoolkit/openvino.genai/blob/3f206d2c50bdd6d6d35ef13c40d90451b1746ef0/text_generation/causal_lm/cpp/chat_sample.cpp</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2062/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>